tpm2/tpm2.go

```diff
@@ -374,7 +395,7 @@ func decodeCreatePrimary(in []byte) (tpmutil.Handle, crypto.PublicKey, error) {
 // CreatePrimary initializes the primary key in a given hierarchy.
 // Second return value is the public part of the generated key.
 func CreatePrimary(rw io.ReadWriter, owner tpmutil.Handle, sel PCRSelection, parentPassword, ownerPassword string, pub Public) (tpmutil.Handle, crypto.PublicKey, error) {
-	cmd, err := encodeCreate(owner, sel, parentPassword, ownerPassword, pub)
+	cmd, err := encodeCreate(owner, sel, parentPassword, ownerPassword, []byte{} /*inSensitive*/, pub)
```

Comment:
You can use nil instead of `[]byte{}` and remove the inline comment

Reply:
the inline comment helps readability by saving the reader a jump to encodeCreate to find out what parameter it represents...why remove it?